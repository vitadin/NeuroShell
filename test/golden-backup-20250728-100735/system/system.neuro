%% System variables test
\set[_echo_command="true"]
\get[#test_mode]