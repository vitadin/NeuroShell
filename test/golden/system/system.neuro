%% System variables test
\get[#test_mode]