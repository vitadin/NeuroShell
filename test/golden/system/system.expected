@user = yaojinsun
#test_mode = true